app/views/admin/communities/edit_welcome_email.haml

```diff
@@ -130,11 +130,20 @@
           %label.sender-address-label= t("admin.communities.outgoing_email.sender_email_label")
           %input.js-sender-email-input{name: "email", type: :text, placeholder: t("admin.communities.outgoing_email.sender_email_placeholder"), disabled: disabled_attr, class: disabled_class}
 
-      .row.js-sender-address-preview-container.sender-address-preview-container.hidden
-        .col-12
-          .sender-address-preview
-            = t("admin.communities.outgoing_email.this_is_how_it_will_look")
-            %span.js-sender-address-preview-values
+      .js-sender-address-preview-container.sender-address-preview-container.hidden
+        .row
+          .col-12
+            .sender-address-preview
+              = t("admin.communities.outgoing_email.this_is_how_it_will_look")
+              %span.js-sender-address-preview-values
+        .row.js-sender-address-preview-container.sender-address-preview-container.hidden
```

Comment:
Seems that there's a small inconsistency between lines 133 and 139: 

```
.js-sender-address-preview-container.sender-address-preview-container.hidden
  .row
```

vs.

```
.row.js-sender-address-preview-container.sender-address-preview-container.hidden
```

I don't know what the classes `.js-sender-address-preview-container.sender-address-preview-container` contain so I don't know if it matters that they are a bit different. Probably it doesn't matter.